tests/behat/features/AgencyComponent.feature

```diff
@@ -12,8 +12,42 @@ Feature: Agency Component Feature
     And for 'Abbreviation' I enter 'TEST'
     And for 'Telephone' I enter '(555) 555-5555'
     And for 'Request Submission Form' I enter 'basic_request_submission_form'
+    And for 'Request Data Year' I enter '2016'
+    And for 'Complex Average Days' I enter '1'
+    And for 'Complex Highest Days' I enter '1'
+    And for 'Complex Lowest Days' I enter 'less than 1'
+    And for 'Complex Median Days' I enter '1'
+    And for 'Expedited Average Days' I enter '3.14'
+    And for 'Expedited Highest Days' I enter '0'
+    And for 'Expedited Lowest Days' I enter '3'
+    And for 'Expedited Median Days' I enter '9'
+    And for 'Simple Average Days' I enter '1'
+    And for 'Simple Highest Days' I enter '6'
+    And for 'Simple Lowest Days' I enter '1'
+    And for 'Simple Median Days' I enter '1'
```

Comment:
From one of the technical requirements in the ticket:
> All processing data fields should be read-only for all users

While these tests do pass, non-admins should only be able to view the data, but should not be able to edit it. We need to update the permissions accordingly. In terms of the tests, they can verify that anonymous users (this implies that agency admins and managers can view it too, so no separate test really needed) can view the data and that non-admins can't edit the data.